app/models/manageiq/providers/amazon/instance_types.rb

```diff
@@ -1,2594 +1,29 @@
-      :architecture            => [:x86_64],
-      :virtualization_type     => [:hvm],
-      :network_performance     => :very_high,
-      :physical_processor      => "Intel Xeon E5-2686 v4",
-      :processor_clock_speed   => "2.3 (base) 2.7 (turbo)",
-      :intel_aes_ni            => true,
-      :intel_avx               => true,
-      :intel_avx2              => true,
-      :intel_turbo             => true,
-      :ebs_optimized_available => true,
-      :enhanced_networking     => true,
-      :cluster_networking      => nil,
-      :vpc_only                => false,
-    },
-
-    "p3.16xlarge" => {
-      :name                    => "p3.16xlarge",
-      :family                  => "Accelerated computing",
-      :description             => "P3 16xlarge",
-      :memory                  => 488.0.gigabytes,
-      :vcpu                    => 64,
-      :ebs_only                => true,
-      :instance_store_size     => 0,
-      :instance_store_volumes  => 0,
-      :architecture            => [:x86_64],
-      :virtualization_type     => [:hvm],
-      :network_performance     => :very_high,
-      :physical_processor      => "Intel Xeon E5-2686 v4",
-      :processor_clock_speed   => "2.3 (base) 2.7 (turbo)",
-      :intel_aes_ni            => true,
-      :intel_avx               => true,
-      :intel_avx2              => true,
-      :intel_turbo             => true,
-      :ebs_optimized_available => true,
-      :enhanced_networking     => true,
-      :cluster_networking      => nil,
-      :vpc_only                => false,
-    },
-
-    "h1.2xlarge"  => {
-      :name                    => "h1.2xlarge",
-      :family                  => "Storage optimized",
-      :description             => "H1 Double Extra Large",
-      :memory                  => 32.0.gigabytes,
-      :vcpu                    => 8,
-      :ebs_only                => false,
-      :instance_store_size     => 2000.0.gigabytes,
-      :instance_store_volumes  => 1,
-      :architecture            => [:x86_64],
-      :virtualization_type     => [:hvm],
-      :network_performance     => :up_to_10_gigabit,
-      :physical_processor      => "",
-      :processor_clock_speed   => "",
-      :intel_aes_ni            => true,
-      :intel_avx               => nil,
-      :intel_avx2              => nil,
-      :intel_turbo             => nil,
-      :ebs_optimized_available => true,
-      :enhanced_networking     => true,
-      :cluster_networking      => nil,
-      :vpc_only                => true,
-    },
-
-    "h1.4xlarge"  => {
-      :name                    => "h1.4xlarge",
-      :family                  => "Storage optimized",
-      :description             => "H1 Quadruple Extra Large",
-      :memory                  => 64.0.gigabytes,
-      :vcpu                    => 16,
-      :ebs_only                => false,
-      :instance_store_size     => 4000.0.gigabytes,
-      :instance_store_volumes  => 2,
-      :architecture   
```

Comment:
No need for the `== true`

Reply:
I have no idea why Github thinks I commented here :/   I commented on the lines that say `== true`

Reply:
done